### DISCUSSION TITLE
Module not found error

### DISCUSSION THREAD

**Discussion by YaohuaLi-Phy** [deepmodeling/dpgen#976] [Q&A] [answered]
Dear developers,

I am new to dpgen, I tried installing dpgen on my ubuntu machine with pip install dpgen and pulling from github, both worked without error 
Successfully installed dpgen-0.10.6
however, testing the installation with dpgen -h gives:

ModuleNotFoundError: No module named 'pymatgen.analysis.defects'

I have pymatgen 2022.9.21 installed, running "from pymatgen.analysis.defects.generators import InterstitialGenerator" also gives the same error. Any suggestions how I should get dpgen to work? Please let me know if more info is needed to solve the problem:)

Many thanks!
Yaohua

**Comment by HuangJiameng** [comment, marked as answer]
This problem has been reported, please follow #963. Before pr963 being merged, you can install pymatgen with version before 2022.7.19 to do autotest normally.

**Reply by YaohuaLi-Phy** [reply]
Thank you very much, Jiameng!
